forge-game/src/main/java/forge/game/phase/PhaseHandler.java

```diff
@@ -559,9 +559,15 @@ private void declareAttackersTurnBasedAction() {
                     }
                 }
 
+                // Exert creatures here
+                List<Card> possibleExerters = CombatUtil.getOptionAttackCostCreatures(whoDeclares);
```

Comment:
my problem there is that it works for Exert now, but you want Enlist later too

Reply:
I have narrowed the result in `hasExertCost`

It will depend if we want to make the choices for both in a single Window
But probably two would be cleaner?

Feel free to extend as needed

Also the rule for multiple Enlist on one creature, I'm not sure how it should happen (in real game)?

Reply:
it isn't about something having both exert or enlist

i just doesn't like it how the function is called possibleExerters when it could be Enlist too